psydac/feec/multipatch/examples_nc/timedomain_maxwell_testcase.py

```diff
@@ -0,0 +1,275 @@
+"""
+    Runner script for solving the time-domain Maxwell problem.
+"""
```

Comment:
same question as above: is there a minimal version of this script in the CI ?

Reply:
See `psydac/feec/multipatch/tests/test_feec_maxwell_multipatch_2d.py`.

Reply:
No, this is currently not tested as we don't have a reference solution.

Reply:
I added a dummy run to the tests.